libs/ledger-live-common/src/families/tron/bridge/js.ts

```diff
@@ -764,8 +764,7 @@ const getTransactionStatus = async (
     if (
       account.type === "TokenAccount" &&
       account.token.tokenType === "trc20" &&
-      energy.eq(0) &&
-      a.spendableBalance.lt(1000000)
+      energy.lt(29650) // temporary value corresponding to usdt trc20 energy
```

Comment:
Why this value ?

Reply:
It's about this price of energy if you are doing a TRC20 USDT transfer ! But yeah it's kind of weird we do not have a simple way to fetch this data. ATM I just take this data from explorer